otj-metrics-actuator/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.opentable.metrics.actuate.micrometer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * {@link ConfigurationProperties @ConfigurationProperties} for Micrometer metrics gateway.
+ * <br><br>
+ * Configuration properties:
+ * <ul>
+ *   <li>{@code management.metrics.export.dw-new.enabled} - To enable/disable export. Disabled by default.</li>
+ *   <li>{@code management.metrics.export.dw-new.prefix} - Prefix to add to the reported metrics. Default is "v2".</li>
```

Comment:
ok so you and @janardhan-chalo have agreed on this? Once I look at JD's PR I'll owonder why these aren't unified or if they are.